src/zendesk.ts

```diff
@@ -550,6 +562,28 @@ namespace Zendesk {
       updated_at?: string
     }
   }
+  export namespace CursorResults {
+    export namespace Sunshine {
+      interface _ {
+        links: {
+          next?: string | null
+          previous?: string | null
+        }
+      }
+      export interface ObjectTypes<Schema> extends _ {
+          data: Zendesk.Sunshine.ObjectType<Schema>[];
```

Comment:
How did semi-colons get in this file? Is prettier not set up for you?

Reply:
Updated my IDE settings, I think I reset it by mistake